src/dbus/server/dbus_thread_object_rcp.cpp

```diff
@@ -1957,6 +1962,29 @@ otError DBusThreadObjectRcp::SetNat64Cidr(DBusMessageIter &aIter)
 }
 #endif // OTBR_ENABLE_NAT64
 
+otError DBusThreadObjectRcp::GetEphemeralKeyEnabled(DBusMessageIter &aIter)
+{
+    otError error = OT_ERROR_NONE;
+
+    VerifyOrExit(DBusMessageEncodeToVariant(&aIter, mBorderAgent.GetEpskcEnabled()) == OTBR_ERROR_NONE,
```

Comment:
`SuccessOrExit` should be more convenient in this use case.

Reply:
done, thanks!